PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerRelEntityMove.java

```diff
@@ -78,28 +78,28 @@ public Entity getEntity(PacketEvent event) {
 		return getEntity(event.getPlayer().getWorld());
 	}
 
-	public int getDx() {
```

Comment:
Might be worth keeping these as returning ints (not like you can do stuff with shorts you couldn't do with ints) and doing a version check to see if you should read from Integers or Shorts